src/meta/expressionEvaluator/functions/validatorSumEqualTo.ts

```diff
@@ -0,0 +1,31 @@
+import { ExpressionFunction } from '@openforis/arena-core/dist/expression/function'
+import { Numbers } from 'utils/numbers'
+import { Objects } from 'utils/objects'
+
+import { NodeValueValidation, NodeValueValidationMessage } from 'meta/assessment'
+
+import { Context } from '../context'
+
+export const validatorSumEqualTo: ExpressionFunction<Context> = {
+  name: 'validatorSumEqualTo',
+  minArity: 2,
+  executor: () => {
+    return (values?: Array<string>, maxValue?: string): NodeValueValidation => {
+      const sum = Numbers.sum(values)
+      const valid = values.some((v) => Objects.isEmpty(v)) || Objects.isEmpty(maxValue) || Numbers.eq(sum, maxValue)
+
+      if (valid) {
+        return { valid }
+      }
+
+      const messages: Array<NodeValueValidationMessage> = [
+        {
+          key: 'generalValidation.sumEqualTo',
```

Comment:
I was looking at this case more closely, how should we change it?
I don't think it makes sense to differentiate each, because we are just checking that the sum === 100